integreat_compass/cms/templates/_base.html

```diff
@@ -11,26 +11,27 @@
                      alt="{% translate "Logo" %}"
                      class="h-24 w-48">
             </a>
+            <a href="{% url 'cms:public:index' %}"
+               class="ml-6 mr-4 hover:text-gray-900">{% translate "All offers" %}</a>
             <a href="#" class="ml-6 mr-4 hover:text-gray-900">{% translate "Dashboard" %}</a>
```

Comment:
Good question. I think you're more up-to-date about this than I am, but I thought the dashboard is a customized view, that depending on your role shows different content. But yes, this means, that at least for not logged in user roles this view is obsolete now. My question is: Should I remove it altogether or rather just make it visible for logged in users?

Reply:
Oh, interesting, so basically the dashboard would be:
- not logged in: All Offers
- board member: Applications to vote on
- integration specialist: All offers
- offer provider: My offers (?)
I guess that makes *some* sense. There is a dedicated Dashboard mockup though:
![image](https://github.com/digitalfabrik/integreat-compass/assets/47758554/1479d560-6d32-4c39-90be-aa3b78421480)
but it's not part of the MVP. So I was under the impression that this would, eventually, become the dashboard for all logged in users, and that until then Dashboard = All Offers.

Reply:
Yes, exactly that are the versions of the dashboard as I understood them too. I didn't realize however that this is not part of the MVP :thinking: Do you agree that for now it would be best to remove this point from the navigation?

Reply:
I think I'd rather remove it for now, then later we can add it as a new "feature" instead of changing what the link does for the users

Reply:
Is the dashboard not part of the MVP?

Reply:
We discussed this, I forgot the result. But I think we said that we do not want to implement the Dashboard and defined the list + filters as the main page?

Reply:
The dashboard was removed as discussed